fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/properties/FDBRecordContextPropertiesTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * FDBRecordContextPropertiesTest.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.provider.foundationdb.properties;
+
+import com.apple.foundationdb.record.RecordCoreException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class FDBRecordContextPropertiesTest {
+    @Test
+    void testDefaultPropertyValue() {
+        final FDBPropertyKey.Builder<Boolean> propertyKeyBuilder = LuceneRecordContextProperties.getLuceneIndexCompressionEnabled();
```

Comment:
Rather than using the Lucene props, it might be a good idea to create test-only properties. That way if the default values for these props are changed, these tests don't break (and then we don't have a dependency on Lucene stuff in `fdb-record-layer-core`)

Reply:
Done.